Makefile

```diff
@@ -5,8 +5,8 @@ wit_files = $(sort $(shell find testdata -name '*.wit' ! -name '*.golden.*'))
 json: $(wit_files)
 
 .PHONY: $(wit_files)
-$(wit_files):
-	wasm-tools component wit -j --all-features $@ > $@.json
+$(wit_files): wit/wasm-tools.wasm
```

Comment:
Let's put the Wazero and wasm-tools dependency into internal/ or x/.

As it stands, this creates a new problem where consumers of package cm will have an implicit dependency on Wazero. We need to figure out a way around this.